modules/learn/pages/views/views-intro.adoc

```diff
@@ -1,5 +1,5 @@
 = Views
-:page-aliases: views:views-intro,understanding-couchbase:views/views-intro,learn:views/sv-ex1-create,learn:views/sv-ex1-query-all.doc,learn:views/sv-ex1-query-area,learn:views/sv-ex1-query-east,learn:views/sv-ex1-query-nonintersect,learn:views/sv-example1,learn:views/sv-example2,learn:views/sv-queries-bbox,learn:views/sv-queries-closed-range,learn:views/sv-queries-open-range,learn:views/sv-query-parameters,learn:views/sv-writing-views-keys,learn:views/sv-writing-views,rest-api:rest-compact-spatialviews,understanding-couchbase:views/sv-ex1-create,understanding-couchbase:views/sv-ex1-query-all,understanding-couchbase:views/sv-ex1-query-area,understanding-couchbase:views/sv-ex1-query-east,understanding-couchbase:views/sv-ex1-query-nonintersect,understanding-couchbase:views/sv-example1,understanding-couchbase:views/sv-example2,understanding-couchbase:views/sv-queries-bbox,understanding-couchbase:views/sv-queries-closed-range,understanding-couchbase:views/sv-queries-open-range,understanding-couchbase:views/sv-query-parameters,understanding-couchbase:views/sv-writing-views-keys,understanding-couchbase:views/sv-writing-views
+:page-aliases: views:views-intro,understanding-couchbase:views/views-intro,learn:views/sv-ex1-create,learn:views/sv-ex1-query-all.doc,learn:views/sv-ex1-query-area,learn:views/sv-ex1-query-east,learn:views/sv-ex1-query-nonintersect,learn:views/sv-example1,learn:views/sv-example2,learn:views/sv-queries-bbox,learn:views/sv-queries-closed-range,learn:views/sv-queries-open-range,learn:views/sv-query-parameters,learn:views/sv-writing-views-keys,learn:views/sv-writing-views,rest-api:rest-compact-spatialviews,understanding-couchbase:views/sv-ex1-create,understanding-couchbase:views/sv-ex1-query-all,understanding-couchbase:views/sv-ex1-query-area,understanding-couchbase:views/sv-ex1-query-east,understanding-couchbase:views/sv-ex1-query-nonintersect,understanding-couchbase:views/sv-example1,understanding-couchbase:views/sv-example2,understanding-couchbase:views/sv-queries-bbox,understanding-couchbase:views/sv-queries-closed-range,understanding-couchbase:views/sv-queries-open-range,understanding-couchbase:views/sv-query-parameters,understanding-couchbase:views/sv-writing-views-keys,understanding-couchbase:views/sv-writing-views,indexes:cb-view-api,indexes:mapreduce-view-replication,indexes:querying-using-spatial-views,architecture:querying-data-with-views
 
 understanding-couchbase:views/sv-ex1-create
```

Comment:
I'm not quite sure exactly what happened here, but I see an orphaned duplicate page alias: `understanding-couchbase:views/sv-ex1-create`